docs/next/guides/columns/column-summary.md

```diff
@@ -14,334 +14,318 @@ tags:
-  let counter = 0;
-
-  const array2d = [...new Array(rows)]
-    .map(_ => [...new Array(columns)]
-    .map(_ => counter++));
+const hot = new Handsontable(container, {
+  licenseKey: 'non-commercial-and-evaluation',
+  data: [
+    [1, 2, 3, 4, 5],
+    [6, 7, 8, 9, 10],
+    [11, 12, 13, 14, 15]
+  ],
+  colHeaders: true,
+  rowHeaders: true,
+  columnSummary: [
+    {
+      sourceColumn: 0,
+      type: 'sum',
+      // set this column summary to display its result in cell (4, 0)
+      destinationRow: 4,
+      destinationColumn: 0
+    },
+    {
+      sourceColumn: 1,
+      type: 'min',
+      // set this column summary to display its result in cell (4, 1)
+      destinationRow: 4,
+      destinationColumn: 1
+    }
+  ]
+});
+```
 
-  if (additionalRows) {
-    array2d.push([]);
-    array2d.push([]);
-  }
+### Step 5: Make room for the destination cell
 
-  return array2d;
-};
+The [`ColumnSummary`](@/api/columnSummary.md) plugin doesn't automatically add new rows to display its summary results.
 
-const container = document.querySelector('#example6');
+So, if you always want to display your column summary result below your existing rows, you need to:
+1. Add an empty row to the bottom of your grid (to avoid overwriting your existing rows).
+2. Reverse row coordinates for your column summary (to always display your summary result at the bottom).
+    ::: tip
+    To reverse row coordinates for your column summary, set the [`reversedRowCoords`](@/api/columnSummary.md#options) option to `true`, and adjust the [`destinationRow`](@/api/columnSummary.md#options) coordinate.
+    :::
 
+```js
 const hot = new Handsontable(container, {
-  data: generateData(5, 5),
-  height: 'auto',
+  licenseKey: 'non-commercial-and-evaluation',
+  data: [
+    [1, 2, 3, 4, 5],
+    [6, 7, 8, 9, 10],
+    [11, 12, 13, 14, 15],
+    // add an empty row
+    [null]
+  ],
   colHeaders: true,
   rowHeaders: true,
-  licenseKey: 'non-commercial-and-evaluation',
   columnSummary: [
     {
-      sourceColumn: 2,
+      sourceColumn: 0,
+      type: 'sum',
+      // for this column summary, count row coordinates from the bottom up
+      reversedRowCoords: true,
+      // now, to always display this column summary in the bottom row,
+      // set `destinationRow` to `0` (i.e. the last possible row)
       destinationRow: 0,
-      destinationColumn: 0,
+      destinationColumn: 0
+    },
+    {
+      sourceColumn: 1,
+      type: 'min',
+      // for this column summary, count row coordinates from the bottom up
       reversedRowCoords: true,
-      type: 'sum',
-      forceNumeric: true
+      // now, to always display this column summary in the bottom row,
+      // set `destinationRow` to `0` (i.e. the last possible row)
+      destinationRow: 0,
+      destinationColumn: 1
     }
   ]
 });
 ```
-:::
 
-## Providing the settings as a function
+## Providing column summary configuration as a function
```

Comment:
The demo in this paragraph (in your code but also in the old version) replaces the first row of data with summaries. I think that we shouldn't do that.

Reply:
@AMBudnik - done: 99430eb. I also added a description and made a few minor changes